src/Entity/Trailer.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Symfony\Component\Serializer\Annotation\Groups;
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\Common\Collections\ArrayCollection;
+use Gedmo\Timestampable\Traits\Timestampable;
+use Gedmo\SoftDeleteable\Traits\SoftDeleteable;
+use Symfony\Component\Validator\Constraints as Assert;
+
+
+/**
+ * @ApiResource(
+ *   attributes={
+ *     "normalization_context"={"groups"={"trailer"}},
+ *     "denormalization_context"={"groups"={"trailer_create"}},
+ *   },
+ *   collectionOperations={
+ *     "get"={
+ *       "method"="GET",
+ *       "access_control"="is_granted('ROLE_DISPATCHER') or is_granted('ROLE_ADMIN')",
```

Comment:
I think `"access_control"="is_granted('ROLE_DISPATCHER')",` will do the same. As ROLE_ADMIN is based on ROLE_DISPATCHER

Reply:
cleaned